plugins/GSdx/GSdx.rc

```diff
@@ -89,13 +89,14 @@ IDB_LOGOGL              BITMAP                  "res\\logo-ogl.bmp"
 // Dialog
 //
 
-IDD_HACKS DIALOGEX 0, 0, 147, 182
+IDD_HACKS DIALOGEX 0, 0, 147, 200
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Hacks Configuration"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    DEFPUSHBUTTON   "OK",IDOK,84,163,56,14
-    GROUPBOX        "USE AT YOUR OWN RISK!",IDC_STATIC,7,7,133,155,0,WS_EX_TRANSPARENT
+    DEFPUSHBUTTON   "OK",IDOK,84,181,56,14
+    GROUPBOX        "USE AT YOUR OWN RISK!",IDC_STATIC,7,7,133,171,0,WS_EX_TRANSPARENT
+	CONTROL         "Preload Data Frame",IDC_PRELOAD_GS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,14,95,80,8
```

Comment:
Use spaces instead of a tab at the start of this line.

Also can you move the "Software Mode Settings" Groupbox in the main dialog up 2 dialog units so that the spacing is more consistent.

I can merge after that's done.